docs: document deprecation (#79)
defi-wonderland/solidity-hardhat-boilerplate: README.md

# Hardhat Boilerplate

# Deprecation notice

> On defi-wonderland we have been using foundry for all new projects for quite some time now, therefore maintaining a template to bootstrap new hardhat projects does not make sense for us anymore. 

Reviewer: ```suggestion
At [Wonderland](https://defi.sucks) we have been using foundry for all new projects for quite some time now, therefore maintaining a template to bootstrap new hardhat projects does not make sense for us anymore. 
```
